Branch-SDK/Branch-SDK/BNCSystemObserver.m

```diff
@@ -73,7 +73,7 @@ + (NSString *)getURIScheme {
 }
 
 + (NSString *)getAppVersion {
-    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleVersion"];
+    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
```

Comment:
Two comments here:

* We may want to keep using the CFBundleVersion, as that refers to the more-descript version identifier -- the build number, instead of the App Store version number. Not sure, open to debate on that
* We should use the constant, if possible. For CFBundleVersion, that's `kCFBundleVersionKey`